### PR TITLE
Refactor manual inline

### PR DESCRIPTION
**Pull request by regiskuckaertz**
This change builds on previous work to rid ourselves of redundant and inefficient code in ads creatives. This time it tackles inline components, the first of which is the manual one.

picture 1

Still to come: books and events.

@Calanthe @JonNorman @rich-nguyen

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_adverts.scss [hunk 1]
```diff
@@ -362,6 +362,50 @@
...
+        > .inline-icon {
+            margin-bottom: $gs-baseline / 2;
+        }
+    }
+
+    .adverts__title {
+        @include fs-header(1);
+    }
+
+    .advert > *:not(.advert__image-container) {
```
Comment (sndrs) [inline static/src/stylesheets/module/commercial/_adverts.scss]: don't need the `*`

Comment (sndrs) [inline static/src/stylesheets/module/commercial/_adverts.scss]: do you need the `.advert >`?

Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts.scss]: Fair enough. The `.advert` is necessary in that case yes.

**Comment by Calanthe** [comment]
👍  Although the `template.js` file is getting bigger and bigger. Maybe at some point it would be nice to reorganise it.

**Comment by regiskuckaertz** [comment]
Right. A looooooot of it I added to normalise the names of properties in the JSON object that is sent from DFP. Sometimes it's `baseUrl`, sometimes `base_url`, sometimes `base__url`. That is maddening 😠 😭

**Pull request merged**